open-source/open-source.md

```diff
@@ -55,22 +70,10 @@ Contributing to Open Source projects offers many benefits:
 
 - [Women Who Code](http://womenwhocode.github.io/#project_reviewers) (See 'Featured Repos' and 'How do we contribute to a project?' Sections)
```

Comment:
Women Who Code shut down 😢 I'm not sure which JS repos would be best to suggest instead. There's a CSS linter I mentioned in one instance where we previously spoke about Operation Code (now also defunct), but I don't really know them well enough to vouch. They seem extremely cool from an external review perspective, but I'm not currently in any JS project opensource communities. @daaimah123 Any suggestions for here?

Reply:
Remove the reference, the others are okay.